tools/maint/create_release.py

```diff
@@ -27,8 +27,12 @@ def update_changelog(release_version, new_version):
   changelog = utils.read_file(changelog_file)
   marker = f'{release_version} (in development)'
   pos = changelog.find(marker)
-  assert pos != -1
-  pos += 2 * len(marker) + 1
+  assert pos >= 0
+  pos += len(marker) + 1
+  # Skip the next line which should just be hypens
+  assert changelog[pos] == '-'
+  pos = changelog.find('\n', pos)
+  assert pos >= 0
```

Comment:
How about `assert pos > last_pos` or something like that?

Reply:
`pos >= 0` is the same thing though.. since we are starting at pos here.